### DISCUSSION TITLE
No colour picker on KDE

### DISCUSSION THREAD

**Discussion by tall-vish** [Schneegans/Burn-My-Windows#454] [Q&A] [answered]
As the title mentions, I can't seem to activate the colour picker on any effect. I have followed the guide on the homepage and installed the `kwidgetaddons-devel` packages. While browsing through other discussions, I ran `journalctl -f -o cat` and got the following error :
`"QFormBuilder was unable to create a custom widget of the class 'KColorButton'; defaulting to base class 'QPushButton'."
`
I am lost at how to proceed on this error, any advice would be helpful.
Running OpenSUSE Tumbleweed, KDE Plasma, Wayland on an AMD gpu. All the latest stuff as well so I do believe I have met the version requirements. 

PS - Also getting a black window after the effect is complete right before the window fades away. Don't if that is a bug as well.
![Screenshot_20241005_040256](https://github.com/user-attachments/assets/a52fd88a-3c23-4f87-8988-6f96084357ec)

**Comment by Schneegans** [comment]
See here: https://github.com/Schneegans/Burn-My-Windows/issues/201

TLDR; If the color picker widgets do not work for you, you may have to install a package called libkf5widgetsaddons-dev (Kubuntu, ...), kwidgetsaddons-devel (Fedora, OpenSuse, ...), or similar, depending on your package manager.

**Reply by tall-vish** [reply]
Hi, I have already installed the `kwidgetaddons-devel` package as mentioned on the github homepage.

**Reply by Schneegans** [reply]
Well, that's weird. It should be part of the [KWidgetsAddons](https://api.kde.org/frameworks/kwidgetsaddons/html/classes.html). Dumb question... did you restart your session?

**Reply by tall-vish** [reply]
Well, I've used Linux for a grand total of 2 weeks now so maybe a dumb answer, I restarted the PC itself, does it count as restarting the session?

**Reply by Schneegans** [reply, marked as answer]
Yeah, this should be sufficient :smile: 

Maybe there is an issue with Plasma 5 / Plasma 6 here. Judging from your screenshot, you have installed version 5.something of the `kwidgetaddons-devel` package. But maybe you are using Plasma 6, for which it seems to be available as well: https://pkgs.chimera-linux.org/package/current/main/x86_64/kwidgetsaddons-devel

However, I have no idea how to get this on your end...

**Reply by tall-vish** [reply]
Hmmm interesting, I'll take a look if there's a similar package available for suse and report back. Thanks!

**Reply by tall-vish** [reply]
You were right! `Kwidgetsaddons` didn't work cos it was the 5.something version. BUT, for some reason, OpenSuSE doesn't have the 6.something package, instead I installed the `KF6-kwidgetaddons-devel` (name is similar to the latter) and now the colour picker works! The black screen issue is gone too after a restart.
